### DISCUSSION TITLE
iOS: Opening repository page closes app

### DISCUSSION THREAD

**Discussion by jamescooke** [community/community#5726] [Mobile]
Using GitHub mobile version 1.32.0 on iOS, loading this repo: https://github.com/jamescooke/openapi-validator-docker/ causes the app to close unexpectedly. It disappears from the list of open apps after closing, so I guess it's crashed.

* Going via Home > Repositories > openapi-validator-docker causes the bug.
* Going via Notifications > Inbox > https://github.com/jamescooke/openapi-validator-docker/pull/86 > then clicking on the repository causes the bug.

All my other personal and work repos load fine.

Is there a known issue, or something on this repo's page that might be causing the problem?

**Comment by jamescooke** [comment]
Bug happens on mobile data and two different wi-fi providers, so I'm pretty sure it's not DNS / content blocking related.

**Reply by alcere** [reply]
Hi @jamescooke ! Can you give it a try again with version 1.33.0 installed and let us know the result, please?

**Reply by jamescooke** [reply]
Hi @alcere , I can't see 1.33.0 in the App Store. Is there something I need to do?

![image](https://user-images.githubusercontent.com/781059/134476514-7af946ca-8e59-47d5-9dff-edfe6ff2726c.png)

**Reply by alcere** [reply]
Sorry, 1.33.0 is only available for Beta users, it will be rolled out to the rest of users next week.

**Reply by jamescooke** [reply]
OK cool - I'll keep an eye and check when 1.33.0 is on general release.

**Reply by jamescooke** [reply]
Can confirm this looks fixed in 1.33.0 🙌🏻
